themes/barbuk/barbuk.theme.bash

```diff
@@ -88,7 +88,14 @@ function _prompt {
         ssh_info="${bold_blue}\u${bold_orange}@${cyan}$host ${bold_orange}in"
     fi
 
-    PS1="\\n${ssh_info} ${purple}$(scm_char)${dir_color}\\w${normal}$(scm_prompt_info)${exit_code}"
+    # Detect python venv
+    if [[ -n "${CONDA_DEFAULT_ENV}" ]]; then
+        python_venv="${CONDA_DEFAULT_ENV}"
```

Comment:
You should probably declare the `python_venv` variable as a `local`, similar to the other variables. Using an undeclared variable might cause warnings/errors.

See line #69 for an example.

Reply:
Thanks for the heads-up - I'll change that.